src/sql/src/plan/statement/ddl.rs

```diff
@@ -1539,8 +1540,14 @@ pub fn plan_create_sink(
 
     let connector_builder = match connector {
         Connector::File { .. } => unsupported!("file sinks"),
-        Connector::Kafka { broker, topic, .. } => kafka_sink_builder(
+        Connector::Kafka {
+            broker,
+            topic,
+            consistency_topic_format,
+            ..
+        } => kafka_sink_builder(
             format,
+            consistency_topic_format,
             &mut with_options,
             broker,
             topic,
```

Comment:
Are you planning on bringing that same check for if the consistency topic is specified twice into here?

Reply:
Yes! I am